### PR TITLE
Add why we chose docusaurus DR

### PR DESCRIPTION
**Pull request by m4rk055**
Related [issue](https://github.com/blindnet-io/communication-management/issues/62).

A part of the [epic](https://github.com/blindnet-io/communication-management/issues/14) to improve the documentation of the blindnet devkit.

After discussion and evaluation of existing docs rendering engines, we decided to go with [Docusaurus](https://docusaurus.io/).
This PR adds the decision record.

### CODE REVIEW

File: decision-records/DR-0001-why-docusaurus.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# Why we use Docusaurus
+
+| Status      | implemented                                                                             |
+| :---------- | :-------------------------------------------------------------------------------------- |
+| **PR #**    | [2](https://github.com/blindnet-io/blindnet-docs/issues/2)                              |
+| **Author**  | Marko Stakic (marko@blindnet.io)                                                        |
+| **Reviwer** | Noël Macé (noel@blindnet.io)                                                            |
```
Comment (noelmace) [inline decision-records/DR-0001-why-docusaurus.md:7]:
```suggestion
| **Author** & **Sponsor** | Marko Stakic (marko@blindnet.io)                                                        |
| **Lead Reviewer** | Noël Macé (noel@blindnet.io)                                                            |
```

See https://github.com/blindnet-io/openness-framework/issues/14 about the sponsor part.

File: decision-records/DR-0001-why-docusaurus.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+## Context and Problem Statement
+
+Blindnet devkit is a project aimed to developers and needs an appropriate documentation. There are multiple OSS engines to render the static documentation websites and we are exploring the best suited one.
```
[the remark this answers is not in the document]
Comment (noelmace) [reply decision-records/DR-0001-why-docusaurus.md]: Perfect, thx :+1: 

FYI, you can just put the commit hash code and Github will autolink it, like that:
41219d73

File: decision-records/DR-0001-why-docusaurus.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+### GitBook
+
+#### pros
+- easy to set-up
+- integration with third party services (Slack, Github, etc.)
+- feature rich
+- beautiful, modern look
+
+#### cons
+- premium product
```
Comment (noelmace) [inline decision-records/DR-0001-why-docusaurus.md:89]: this may require a bit more information :wink:

**Review by noelmace — COMMENTED** [review]
LGTM, except for the Gitbook cons ("premier product" sounds vague and well, not so much of a const to me :smile:).

File: decision-records/DR-0001-why-docusaurus.md (hunk 3 above)
Comment (m4rk055) [reply decision-records/DR-0001-why-docusaurus.md]: Updated in c15ef210849ea037b8c269c43425bd5046a9e72d

**Pull request merged**